crates/bevy_reflect/src/impls/std.rs

```diff
@@ -620,6 +622,220 @@ impl_type_path!(::bevy_utils::hashbrown::hash_map::DefaultHashBuilder);
 impl_type_path!(::bevy_utils::NoOpHash);
 impl_type_path!(::bevy_utils::hashbrown::HashMap<K, V, S>);
 
+macro_rules! impl_reflect_for_btree_map {
```

Comment:
Was this macro really necessary? It's not like it will be used more than once, and it makes source navigation harder.